src/main/java/dev/galacticraft/impl/universe/position/config/SatelliteConfig.java

```diff
@@ -54,8 +56,8 @@ public final class SatelliteConfig implements CelestialBodyConfig {
             LevelStem.CODEC.fieldOf("dimension_options").forGetter(SatelliteConfig::dimensionOptions)
     ).apply(instance, SatelliteConfig::new));
 
-    private final ResourceKey<CelestialBody<?, ?>> parent;
-    private final ResourceKey<Galaxy> galaxy;
+    private final Optional<ResourceKey<CelestialBody<?, ?>>> parent;
```

Comment:
Satellites should always have a parent body

Reply:
I was thinking of rouge satellites not attached to any celestial body.